### ISSUE TITLE
Version 6.0.0 Crashes when the AC is off

### ISSUE THREAD

**Issue by rupertbg** [codyc1515/homebridge-panasonic-air-conditioner#58]
```
TypeError: Cannot read property 'INACTIVE' of undefined
    at PanasonicAC.<anonymous> (/homebridge/node_modules/homebridge-panasonic-air-conditioner/index.js:290:99)
    at Request.self.callback (/homebridge/node_modules/homebridge-panasonic-air-conditioner/node_modules/request/request.js:185:22)
    at Request.emit (events.js:315:20)
    at Request.<anonymous> (/homebridge/node_modules/homebridge-panasonic-air-conditioner/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:315:20)
    at IncomingMessage.<anonymous> (/homebridge/node_modules/homebridge-panasonic-air-conditioner/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:421:28)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by arkadicolson** [comment]
The same issue here...

Arkadi

**Comment by codyc1515** [comment]
That’s strange. Admittedly I forgot to test this. Should be fixed now in 6.0.1
